test/e2e_node/cpu_manager_test.go

```diff
@@ -415,7 +415,7 @@ func runCPUManagerTests(f *framework.Framework) {
 }
 
 // Serial because the test updates kubelet configuration.
-var _ = framework.KubeDescribe("CPU Manager [Serial]", func() {
+var _ = framework.KubeDescribe("CPU Manager [Feature:CPUManager] [Serial]", func() {
```

Comment:
Remove the `[Serial]` tag so that the serial job won't run it. If we decide to add it back later, we can change the flag.

Reply:
Ok. That makes sense. I've updated the PR.